src/components/bills/index.tsx

```diff
@@ -79,44 +107,63 @@ export default function UploadBillForm() {
   const submitUploadBillForm = async () => {
     setLoading(true);
     try {
-      if (!data["pdfFile"].name) {
-        toast.error("Please upload a valid PDF file");
+      if (!data["pdfFiles"] || data["pdfFiles"].length === 0) {
+        toast.error("Please upload valid PDF files");
         setLoading(false);
         return;
       }
 
-      const billUrl = await storeBillPdf(data["pdfFile"], data);
+      const pdfFiles = await convertImagesToSeparatePdfs(data["pdfFiles"]);
 
-      if (!billUrl) {
-        toast.error("Error uploading the bill file");
+      if (pdfFiles.length === 0) {
+        toast.error("Error converting files to PDFs");
+        setLoading(false);
         return;
       }
-      const notionObject = new Bill({
-        Name: `${boatInfo["Nombre"]}-${moment(data["Date"]).format(
-          "DD-MM-YY"
-        )}`,
-        Date: data["Date"],
-        Amount: +data["Amount"],
-        Boat: data["Boat"],
-        Type: data["Type"],
-        Bill: billUrl,
-      });
-
-      const res = await createBillRecord(notionObject);
-      if (!res) {
-        toast.error("Failed to create the record");
-        return;
+
+      const fileIds: string[] = [];
+
+      for (const pdfFile of pdfFiles) {
+        const billUrl = await storeBillPdf(pdfFile, data);
+
+        if (!billUrl) {
+          toast.error(`Error uploading the file: ${pdfFile.name}`);
+          continue;
+        }
+
+        fileIds.push(billUrl);
+
+        const notionObject = new Bill({
+          Name: `${boatInfo["Nombre"]}-${moment(data["Date"]).format(
+            "DD-MM-YY"
+          )}`,
+          Date: data["Date"],
+          Amount: +data["Amount"],
+          Boat: data["Boat"],
+          Type: data["Type"],
+          Bill: billUrl,
+        });
```

Comment:
This is wrong

No need 3 bills in notion

is just one bill with 3 attachments